Cargo.toml

```diff
@@ -1,23 +1,22 @@
 [package]
 name = "lazymc"
-version = "0.2.6"
+version = "0.2.8"
```

Comment:
Change this back to `0.2.6`. I'll bump the version on release from `master`.

Reply:
Oh I used `cargo add` and I guess it just threw it in wherever